src/migrations/20250127163224-update-eclkc-maps-to.js

```diff
@@ -0,0 +1,81 @@
+const { prepMigration } = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+
+      queryInterface.addColumn('Resources', 'mapsTo', { type: Sequelize.INTEGER, allowNull: true }, { transaction });
```

Comment:
await queryInterface.addColumn(
  'Resources',
  'mapsTo',
  {
    type: Sequelize.INTEGER,
    allowNull: true,
    references: {
      model: 'Resources', // Table name
      key: 'id', // Column to reference
    },
    onUpdate: 'CASCADE',
    onDelete: 'SET NULL', // Adjust as needed (e.g., 'CASCADE' or 'RESTRICT')
  },
  { transaction }
);

Reply:
done.